test-boot.mk

```diff
@@ -1,35 +1,31 @@
 
-BOOT_NAME=boot
+BOOT_NAME = boot
 
-base-files=
-base-files+=$(wildcard test/mexpr/*.mc)
-base-files+=$(wildcard test/mlang/*.mc)
-base-files+=$(wildcard stdlib/mexpr/*.mc)
-base-files+=$(wildcard stdlib/c/*.mc)
-base-files+=$(wildcard stdlib/ad/*.mc)
-base-files+=$(wildcard stdlib/parser/*.mc)
+base-files += $(wildcard test/mexpr/*.mc)
+base-files += $(wildcard test/mlang/*.mc)
+base-files += $(wildcard stdlib/mexpr/*.mc)
+base-files += $(wildcard stdlib/c/*.mc)
+base-files += $(wildcard stdlib/ad/*.mc)
+base-files += $(wildcard stdlib/parser/*.mc)
+base-files += $(wildcard stdlib/*.mc)
 
-stdlib-root-files=$(wildcard stdlib/*.mc)
+sundials-files += $(wildcard test/sundials/*.mc)
+sundials-files += $(wildcard stdlib/sundials/*.mc)
```

Comment:
The sundials implementation should not be tested with boot since they are "compiled only" now so those files should be removed.

Reply:
Done!

Reply:
Thx!